### ISSUE TITLE
github.com/akavel/rsrc dependency has removed coff.NewRDATA

### ISSUE THREAD

**Issue by deoxxa** [GeertJohan/go.rice#162]
Looks like the embed-syso mode will need to be updated or removed.

**Comment by SnorlaxGengar** [comment]
It's seems that been removed at https://github.com/akavel/rsrc/commit/6c59b92bada085ce1b7ad670435ffe594b4a2656

**Comment by prologic** [comment]
This is also affecting https://github.com/jointwt/twtxt/issues/290

**Comment by GeertJohan** [comment]
As a quick-fix I will remove embed-syso until we can properly fix it. So that users of other embed variants are not affected.

**Comment by prologic** [comment]
> As a quick-fix I will remove embed-syso until we can properly fix it. So that users of other embed variants are not affected.

Please remind me what the default is again? I don't recall using any special flag for my projects that use `rice-embed`, but need to be sure things _won't_ horribly break :/

**Comment by GeertJohan** [comment]
The default is embed-go, embed-syso has always been experimental. If you're not calling it explicitly, you won't be affected.

**Comment by GeertJohan** [comment]
embed-syso has been disabled as of https://github.com/GeertJohan/go.rice/commit/3f72e1a84580591c956f602330f0719e8b422935

**Comment by prologic** [comment]
Sounds good to me! Let's nuke it then and move on with our lives :D

**Comment by akavel** [comment]
Hi! I'm the author of rsrc. I had no idea you're using rsrc, and NewRDATA in particular 😅 If that's some kind of real practical usage, I'm quite open to bringing it back - however I'd need some reasonable testcases. In my project, the feature using NewRDATA is basically defunct since Go 1.4. Which is *reeeaaaaaallllllyyyyyy* old now. That's basically why I decided to remove it, when I had to do some wider changes in the codebase - as I didn't have a way to test whether making changes to this dead code didn't break it.

At this moment, from the discussion above, I seem to notice that was deemed "experimental feature" in your case, and you don't seem to be missing it super strong. Do I read it right? Or is it important enough to you that you'd like to discuss what would it take to bring the feature back? :)

Cheers,
/M.
